Migration.md

```diff
@@ -5,6 +5,13 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo Sim 8.x to 9.0
+
+ * **Modified**:
+   + In the Physics system, all `*VelocityCmd` components are deleted after
```

Comment:
mention that previously it was setting them to zero?

Reply:
https://github.com/gazebosim/gz-sim/pull/2228/commits/f4ffd4a8721915a8c9ac90ab45006927835b7bf3